src/alchemlyb/parsing/amber.py

```diff
@@ -65,8 +65,9 @@ def __init__(self, filename):
         self.filename = filename
         try:
             self.fileh = anyopen(self.filename, 'r')
-        except Exception as ex:  # pragma: no cover
+        except Exception as ex:
```

Comment:
We don't need to alias as `ex`
```suggestion
        except:
```

If in doubt, less code is better ;-).